src/TraceEvent/TraceLog.cs

```diff
@@ -10632,10 +10831,7 @@ public static TraceProcess Process(this TraceEvent anEvent)
                 throw new InvalidOperationException("Attempted to use TraceLog support on a non-TraceLog TraceEventSource.");
             }
             TraceProcess ret = log.Processes.GetProcess(anEvent.ProcessID, anEvent.TimeStampQPC);
-            // When the trace was converted, a TraceProcess should have been created for
-            // every mentioned Process ID.
-            // When we care, we should ensure this is true for the RealTime case.
-            Debug.Assert(ret != null || log.IsRealTime);
+            Debug.Assert(ret != null);
```

Comment:
It looks like there are a couple of these asserts that were changed to remove `log.IsRealTime`.  Can you tell me more about what changed here?

Reply:
Gotcha.  Go ahead and remove it then.  If we notice this isn't the case, we can always re-add it.